### ISSUE TITLE
Compiler error using items with id and Polyhedron_3

### ISSUE THREAD

**Issue by hstruebe** [CGAL/cgal#1683]
When using the [Polyhedron_items_with_id_3](https://github.com/CGAL/cgal/blob/f4a29597e4dd30f569eadab7cea6c395331b8b67/Polyhedron/include/CGAL/Polyhedron_items_with_id_3.h) model as Items in [Polyhedron_3]( https://github.com/CGAL/cgal/blob/f4a29597e4dd30f569eadab7cea6c395331b8b67/Polyhedron/include/CGAL/Polyhedron_3.h ) and then using the _set_halfedgeds_items_id_ function from Polyhedron_items_with_id_3 it [fails](http://pastebin.com/8f4j4BTL).

The Error is because [HalfedgeDS_list.h](https://github.com/CGAL/cgal/blob/f4a29597e4dd30f569eadab7cea6c395331b8b67/HalfedgeDS/include/CGAL/HalfedgeDS_list.h) uses _faces_begin()_ and Polyhedron_items_with_id_3 expected _facets_begin()_.

I'm noot deep into CGAL so i dont know if this is simple the way it is or it's actually a bug.
Example:
```c++
#include <CGAL/Polyhedron_3.h>
#include <CGAL/Simple_cartesian.h>

int main(int argc, char const *argv[]) {
  CGAL::Polyhedron_3<Kernel, CGAL::Polyhedron_items_with_id_3> P;
  P.make_tetrahedron();
  CGAL::set_halfedgeds_items_id(P.hds());
  return 1;
}
```
Greetings

**Comment by sloriot** [comment]
You should use `CGAL::set_halfedgeds_items_id(P);`. I've submitted a PR fixing the doc.

**Comment by lrineau** [comment]
The documentation bug has been fixed by #1684, merged in the `master` Git branch. It will be in CGAL-4.10.
